### ISSUE TITLE
New error when running main

### ISSUE THREAD

**Issue by AurelienFT** [hawolt/league-client-api#7]
Today wanted to experiment a bit using your package I ran into a security page trigger from cloudfare : 
```
[22 Aug 2023 21:27:18] [INTERNAL] Unable to locate log.properties in default directory
[22 Aug 2023 21:27:18] [DEBUG] [cache] store: (k:RiotClientFoundation.dll, v:70.0.0.247)
[22 Aug 2023 21:27:19] [DEBUG] [cache] store: (k:RiotGamesApi.dll, v:23.8.0.1382)
[22 Aug 2023 21:27:19] [INFO] RiotClient/70.0.0.247.1382 rso-auth (Windows;10;;Professional, x64)
[22 Aug 2023 21:27:19] [DEBUG] [http] 
POST https://auth.riotgames.com/api/v1/authorization
accept: application/json
content-type: application/json
pragma: no-cache
user-agent: RiotClient/70.0.0.247.1382 rso-auth (Windows;10;;Professional, x64)
{"acr_values":"","scope":"openid link ban lol_region lol summoner offline_access","claims":"","response_type":"token id_token","code_challenge_method":"","redirect_uri":"http://localhost/redirect","nonce":"ZGI0MzlhN2ItMTc0Ni00ND","code_challenge":"","client_id":"riot-client"}
cache-control: max-age=15
cf-ray: 7fad8331b80ad598-CDG
content-type: text/html; charset=UTF-8
date: Tue, 22 Aug 2023 19:27:19 GMT
expires: Tue, 22 Aug 2023 19:27:34 GMT
server: cloudflare
set-cookie: __cf_bm=yZfYngLej9qlE0rMjx1xcDIuApDK4ac1oHLS9ZuDZ2M-1692732439-0-AXmWeaItNEG56ZjaM1xMHMs43L90M5UcPc+EJkHYyaxxffnXp3mFg1PjesX3DuCO/za3fD4PTMK0VFoZPOACutc=; path=/; expires=Tue, 22-Aug-23 19:57:19 GMT; domain=.riotgames.com; HttpOnly; Secure; SameSite=None
vary: Accept-Encoding
x-frame-options: SAMEORIGIN
x-riotgames-cdn: Cloudflare
<!DOCTYPE html>
<!--[if lt IE 7]> <html class="no-js ie6 oldie" lang="en-US"> <![endif]-->
<!--[if IE 7]>    <html class="no-js ie7 oldie" lang="en-US"> <![endif]-->
<!--[if IE 8]>    <html class="no-js ie8 oldie" lang="en-US"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en-US"> <!--<![endif]-->
<head>
<title>Attention Required! | Cloudflare</title>
<meta charset="UTF-8" />
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<meta http-equiv="X-UA-Compatible" content="IE=Edge" />
<meta name="robots" content="noindex, nofollow" />
<meta name="viewport" content="width=device-width,initial-scale=1" />
<link rel="stylesheet" id="cf_styles-css" href="/cdn-cgi/styles/cf.errors.css" />
<!--[if lt IE 9]><link rel="stylesheet" id='cf_styles-ie-css' href="/cdn-cgi/styles/cf.errors.ie.css" /><![endif]-->
<style>body{margin:0;padding:0}</style>


<!--[if gte IE 10]><!-->
<script>
  if (!navigator.cookieEnabled) {
    window.addEventListener('DOMContentLoaded', function () {
      var cookieEl = document.getElementById('cookie-alert');
      cookieEl.style.display = 'block';
    })
  }
</script>
<!--<![endif]-->


</head>
<body>
  <div id="cf-wrapper">
    <div class="cf-alert cf-alert-error cf-cookie-error" id="cookie-alert" data-translate="enable_cookies">Please enable cookies.</div>
    <div id="cf-error-details" class="cf-error-details-wrapper">
      <div class="cf-wrapper cf-header cf-error-overview">
        <h1 data-translate="block_headline">Sorry, you have been blocked</h1>
        <h2 class="cf-subheadline"><span data-translate="unable_to_access">You are unable to access</span> riotgames.com</h2>
      </div><!-- /.header -->

      <div class="cf-section cf-highlight">
        <div class="cf-wrapper">
          <div class="cf-screenshot-container cf-screenshot-full">
            
              <span class="cf-no-screenshot error"></span>
            
          </div>
        </div>
      </div><!-- /.captcha-container -->

      <div class="cf-section cf-wrapper">
        <div class="cf-columns two">
          <div class="cf-column">
            <h2 data-translate="blocked_why_headline">Why have I been blocked?</h2>

            <p data-translate="blocked_why_detail">This website is using a security service to protect itself from online attacks. The action you just performed triggered the security solution. There are several actions that could trigger this block including submitting a certain word or phrase, a SQL command or malformed data.</p>
          </div>

          <div class="cf-column">
            <h2 data-translate="blocked_resolve_headline">What can I do to resolve this?</h2>

            <p data-translate="blocked_resolve_detail">You can email the site owner to let them know you were blocked. Please include what you were doing when this page came up and the Cloudflare Ray ID found at the bottom of this page.</p>
          </div>
        </div>
      </div><!-- /.section -->

      <div class="cf-error-footer cf-wrapper w-240 lg:w-full py-10 sm:py-4 sm:px-8 mx-auto text-center sm:text-left border-solid border-0 border-t border-gray-300">
  <p class="text-13">
    <span class="cf-footer-item sm:block sm:mb-1">Cloudflare Ray ID: <strong class="font-semibold">7fad8331b80ad598</strong></span>
    <span class="cf-footer-separator sm:hidden">&bull;</span>
    <span id="cf-footer-item-ip" class="cf-footer-item hidden sm:block sm:mb-1">
      Your IP:
      <button type="button" id="cf-footer-ip-reveal" class="cf-footer-ip-reveal-btn">Click to reveal</button>
      <span class="hidden" id="cf-footer-ip">91.167.142.203</span>
      <span class="cf-footer-separator sm:hidden">&bull;</span>
    </span>
    <span class="cf-footer-item sm:block sm:mb-1"><span>Performance &amp; security by</span> <a rel="noopener noreferrer" href="https://www.cloudflare.com/5xx-error-landing" id="brand_link" target="_blank">Cloudflare</a></span>
    
  </p>
  <script>(function(){function d(){var b=a.getElementById("cf-footer-item-ip"),c=a.getElementById("cf-footer-ip-reveal");b&&"classList"in b&&(b.classList.remove("hidden"),c.addEventListener("click",function(){c.classList.add("hidden");a.getElementById("cf-footer-ip").classList.remove("hidden")}))}var a=document;document.addEventListener&&a.addEventListener("DOMContentLoaded",d)})();</script>
</div><!-- /.error-footer -->


    </div><!-- /#cf-error-details -->
  </div><!-- /#cf-wrapper -->

  <script>
  window._cf_translation = {};
  
  
</script>

</body>
</html>
```

This cause this error in the Java code: 
```
org.json.JSONException: A JSONObject text must begin with '{' at 1 [character 2 line 1]
    at org.json.JSONTokener.syntaxError (JSONTokener.java:497)
    at org.json.JSONObject.<init> (JSONObject.java:208)
    at org.json.JSONObject.<init> (JSONObject.java:404)
    at com.hawolt.virtual.riotclient.instance.AbstractVirtualRiotClientInstance.get (AbstractVirtualRiotClientInstance.java:74)
    at com.hawolt.virtual.riotclient.instance.AbstractVirtualRiotClientInstance.getRiotClientSupplier (AbstractVirtualRiotClientInstance.java:118)
    at com.hawolt.virtual.riotclient.instance.AbstractVirtualRiotClientInstance.login (AbstractVirtualRiotClientInstance.java:111)
    at com.mycompany.app.App.main (App.java:34)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:104)
    at java.lang.reflect.Method.invoke (Method.java:577)
    at org.codehaus.mojo.exec.ExecJavaMojo$1.run (ExecJavaMojo.java:282)
    at java.lang.Thread.run (Thread.java:1496)
```

**Comment by slyk26** [comment]
TLDR: riot changed some settings on the mighty cloudflare wall  to prevent us from accessing their api. 

my first guess would be the user agent or some other "Swift Rift" Meme that they saw some anomalies on their dashboard, worst case some ban

**Comment by hawolt** [comment]
should be resolved
